apisix/plugins/ai.lua

```diff
@@ -280,7 +281,8 @@ end
 
 
 function _M.init_worker()
-    orig_router_match = router.router_http.match
+    orig_router_match = base_router.match_uri
```

Comment:
we need a better name for `orig_router_match`

Reply:
done